::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 30 Jan 2018 23:12:52 +0100
Subject: [PATCH] linux:system:repo:mcp: Pin to 2018-01-15-100123

Recent changes in MCP repos lead to breakage on ODL scenarios.
Pin said repos to the latest known working snapshot.
Since the pinning mechanism is not yet fully upstream in
salt-formula-linux, hard set the snapshot URL via this patch.

JIRA: ARMBAND-355

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/repo/mcp/openstack.yml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/linux/system/repo/mcp/openstack.yml b/linux/system/repo/mcp/openstack.yml
index 22defb7..0579449 100644
--- a/linux/system/repo/mcp/openstack.yml
+++ b/linux/system/repo/mcp/openstack.yml
@@ -6,9 +6,9 @@ parameters:
     system:
       repo:
         mirantis_openstack:
-          source: "deb http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/${_param:linux_system_codename} ${_param:openstack_version} main"
+          source: "deb http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/snapshots/xenial-2018-01-15-100123 ${_param:openstack_version} main"
           architectures: amd64
-          key_url: "http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/${_param:linux_system_codename}/archive-mcp${_param:openstack_version}.key"
+          key_url: "http://mirror.fuel-infra.org/mcp-repos/${_param:openstack_version}/snapshots/xenial-2018-01-15-100123/archive-mcp${_param:openstack_version}.key"
           pin:
           - pin: 'release l=${_param:openstack_version}'
             priority: 1100
